### PR TITLE
fix(root): fixed broken workflow

### PR DESCRIPTION
**Pull request by GCHQ-Developer-741**
Fixed on trigger for deleting folder workflow

### CODE REVIEW

**Review by GCHQ-Developer-112 — DISMISSED** [review]

**Review by GCHQ-Developer-530 — DISMISSED** [review]

File: .github/workflows/add-qa-link-to-issues.yml [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
     runs-on: ubuntu-latest
     permissions:
-      issues: write
+      pull-requests: write
     steps:
-      - name: Add comment
-        uses: peter-evans/create-or-update-comment@a35cf36e5301d70b76f316e867e7788a55a31dae
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_HEAD_REF#refs/heads/})"
```
Comment (jd239) [inline .github/workflows/add-qa-link-to-issues.yml:14]: I've recently seen warnings that set-output will be deprecated soon. https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/.

File: .github/workflows/delete-gh-pages-folder.yml [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_HEAD_REF#refs/heads/})"
+        id: extract_branch
+
       - name: Checkout
         uses: actions/checkout@v3
         with:
           repository: mi6/ic-design-system-githubpages
 
       - name: Remove and commit
```
Comment (jd239) [inline .github/workflows/delete-gh-pages-folder.yml:19]: Can we add a commit message to this step so we have a record of what's getting deleted?

**Review by jd239 — CHANGES_REQUESTED** [review]
Nice workflows. A couple minor comments.

File: .github/workflows/delete-gh-pages-folder.yml (hunk 2 above)
Comment (GCHQ-Developer-741) [reply .github/workflows/delete-gh-pages-folder.yml:18]: Have added a commit message

**Review by jd239 — APPROVED** [review]

**Review by GCHQ-Developer-530 — APPROVED** [review]

**Review by GCHQ-Developer-847 — APPROVED** [review]

**Pull request merged**